ingest_insarmaps.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
```

Comment:
This is a new script and the function run_insar_maps from _process_utilities is replaced with this script